packages/commerce-sdk-react/src/provider.tsx

```diff
@@ -66,7 +67,8 @@ const CommerceApiProvider = (props: CommerceApiProviderProps): ReactElement => {
         siteId,
         shortCode,
         locale,
-        currency
+        currency,
+        fetchedToken
```

Comment:
this is not being used, i think you forgot to pass it in when instantiating the Auth class, line 111.

Reply:
Good catch. Fixed